docusaurus/sidebars.js

```diff
@@ -546,6 +546,7 @@ module.exports = {
         "operator-guides/browsing-output-logs",
         "operator-guides/clear",
         "operator-guides/refreshes",
+        "operator-guides/resumability",
```

Comment:
Not sure it's relevant to this section since you're not really editing/managing this when you're using Airbyte. Should it be under "Understanding Airbyte"?